### ISSUE TITLE
Check which species codes should be added to WoRMS, EURING and NT

### ISSUE THREAD

**Issue by peterdesmet** [ices-tools-dev/esas#60]
NT = non-taxonomical units, see https://vocab.ices.dk/?CodeID=79748

**Comment by peterdesmet** [comment]
After investigation, we find it a lot more straightforward if data providers can **upload data using a single ESAS code**, rather than split EURING/NT code concepts. The ESAS codes have been used for over 3 decades and will not change. For the submission of new taxa not on the list, data providers can use Aphia IDs from WoRMS, which is actively maintained.

We have completed the mapping exercise and **every ESAS code now has an associated Aphia ID** (including newly requested ones). For completeness, we have also linked with EURING codes were one is available: https://ices-tools-dev.github.io/esas/species/

We suggest that for data downloads (not uploads) the original provided code (whether ESAS or Aphia ID) is returned, **as well as the associated Aphia ID**. Returning the scientific and English name would be useful as well. Note that some specific codes (e.g. `127` Podiceps grisegena / cristatus, `128` Podiceps auritus / nigricollis, `129` Podiceps sp.) share the same Aphia ID (`137056` Podiceps). By returning the original code _and_ Aphia ID, analyses are possible at the (original) granular level _and_ the overarching level using WoRMS as the reference list.
